demo/java/ImageClassification.java

```diff
@@ -0,0 +1,55 @@
+import mmdeploy.Classifier;
+import mmdeploy.PixelFormat;
+import mmdeploy.DataType;
+import mmdeploy.Mat;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.awt.image.DataBufferByte;
+import java.io.File;
+import java.io.IOException;
+
+public class ImageClassification {
+
+    private static Mat loadImage(String path) throws IOException {
+        BufferedImage img = ImageIO.read(new File(path));
+        byte[] data = ((DataBufferByte) img.getData().getDataBuffer()).getData();
+        return new Mat(img.getHeight(), img.getWidth(), img.getColorModel().getNumComponents(),
+                PixelFormat.BGR, DataType.INT8, data);
+    }
+
+    public static void main(String[] args) {
+        // Parse arguments
+        if (args.length != 3) {
+            System.out.println("usage:\njava ImageClassification modelPath deviceName imagePath");
+            return;
+        }
+        String modelPath = args[0];
```

Comment:
Let's keep the sequence of arguments the same with c demo, i.e., `deviceName`, `modepPath` and `imagePath`